plenum/test/malicious_behaviors_node.py

```diff
@@ -68,6 +60,23 @@ def dont_send_prepare_and_commit_to(nodes, ignore_node_name):
         node.send = types.MethodType(dont_send_prepare_commit, node)
 
 
+def dont_send_propagate(self, msg: Any, *rids, signer=None, message_splitter=None):
```

Comment:
Can we use already existent functionality for this:
```
    A, B, C, D = txnPoolNodeSet
    delay(Propagate, frm=[C, D], to=A, howlong=howlong)
```

Reply:
yes